src/Intl/Idn/Info.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Intl\Idn;
+
+/**
+ * Copyright (c) 2020 Trevor Rowbotham
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * Originally forked from
+ * https://github.com/TRowbotham/idna/tree/0.1.0
+ *
+ * @author Trevor Rowbotham <trevor.rowbotham@pm.me>
+ */
+class Info
+{
+    /**
+     * @var bool
+     */
+    private $bidiDomain;
+
+    /**
+     * @var int
+     */
+    private $errors;
+
+    /**
+     * @var bool
+     */
+    private $validBidiDomain;
+
+    /**
+     * @var bool
+     */
+    private $transitionalDifferent;
+
+    public function __construct()
+    {
+        $this->bidiDomain = false;
+        $this->errors = 0;
+        $this->transitionalDifferent = false;
+        $this->validBidiDomain = true;
```

Comment:
these should be set as default values on the properties instead of adding a constructor.
that's also allow removing the docblocks on properties.
BTW, since all this is internal, we could even make the properties public and remove all methods on the class.

Reply:
Done